packages/webpack/src/utils/style-loader.js

```diff
@@ -64,32 +68,25 @@ export default class StyleLoader {
   }
 
   css(options) {
-    if (this.isServer && this.extractCSS) {
-      options.exportOnlyLocals = true
-    }
-    return {
-      loader: 'css-loader',
-      options
-    }
+    options.exportOnlyLocals = this.exportOnlyLocals
```

Comment:
It would be more better to make options immutable. But there are other parts that need such refactor. So just ignore this comment for this PR :D